app/ledger.js

```diff
@@ -263,19 +269,23 @@ const doAction = (action) => {
 /*
  * module entry points
  */
+
 var init = () => {
   try {
     appDispatcher.register(doAction)
     initialize(getSetting(settings.PAYMENTS_ENABLED))
 
-    doneTimer = setInterval(doneWriter, 1 * msecs.hour)
+    doneTimer = setInterval(doneWriter, 1 * msecs.minute)
```

Comment:
(I talked with @mrose17; it's fine as original or as 1 minute, but he advised 1 minute is optimal)